src/components/choropleth/tooltips/region/escalation-tooltip.tsx

```diff
@@ -25,29 +23,29 @@ export const escalationTooltip = (router: NextRouter) => {
       );
     };
 
+    const escalationText = (text.escalatie_niveau.types as Record<
+      string,
+      unknown
+    >)[level] as { titel: string; valid_from: string };
```

Comment:
isn't it possible to write this as
```suggestion
    const escalationText = (text.escalatie_niveau.types as Record<
      string,
      { titel: string; valid_from: string }
    >)[level];
```

Reply:
I gave it a try. TS says no

Reply:
This worked

```
const escalationText = ((text.escalatie_niveau.types as unknown) as Record<
      EscalationLevel,
      { titel: string; valid_from: string }
    >)[level];
```